README.md

```diff
@@ -34,6 +34,10 @@ Use terminal, iTerm, etc. Not XTerm.
         - _**With**_ remote user specified
             - `sh runAsUser.sh bjensen`
                 - `bjensen` is a user included in the test data.  Others are available.  For the purposes of the HWF event, use only known users.
+        - _**With gunicorn**_ and remote user specified
+            - `REMOTE_USER=bjensen gunicorn --workers=1 --bind=0.0.0.0:8000 hacks_mbof.wsgi:application`
```

Comment:
Also, it's a fake user with a reserved uniqname.  Historically, `bjensen` was the username found in ITD/ITCS documentation, and her uniqname was reserved.  She no longer appears in MCommunity (sadly), however and AFAIK, nobody can authN as her.  However, the uniqname will not be given out to anybody in the future, either.